index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Baroka'a Portofolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <img class="logo" src="images/image_geometry.png" alt="logo">
```

Comment:
- [ ] Kindly wrap the logo inside of an `a` tag. This can be used as a quick access link later to go to the home page and is considered a good practice.